napari/layers/points/_tests/test_points.py

```diff
@@ -2661,3 +2661,21 @@ def test_docstring():
     validate_all_params_in_docstring(Points)
     validate_kwargs_sorted(Points)
     validate_docstring_parent_class_consistency(Points)
+
+
+@pytest.mark.parametrize(
+    'key',
+    [
+        'edge_width',
+        'edge_width_is_relative',
+        'edge_color',
+        'edge_color_cycle',
+        'edge_colormap',
+        'edge_contrast_limits',
+    ],
+)
+def test_as_layer_data_tuple_read_deprecated_attr(key: str):
+    layer = Points()
+    _, attrs, _ = layer.as_layer_data_tuple()
+    with pytest.warns(FutureWarning):
```

Comment:
```suggestion
    with pytest.warns(FutureWarning, match="is deprecated since"):
```
Exact string may need to be tested, but it is much better to validate text of warning not to catch wrong warning.